build_tools/travis/install.sh

```diff
@@ -53,11 +53,14 @@ if [[ "$DISTRIB" == "conda" ]]; then
     if [[ "$INSTALL_MKL" == "true" ]]; then
         conda create -n testenv --yes python=$PYTHON_VERSION pip nose \
             numpy=$NUMPY_VERSION scipy=$SCIPY_VERSION numpy scipy \
-            cython=$CYTHON_VERSION libgfortran mkl flake8
+            cython=$CYTHON_VERSION libgfortran mkl flake8 \
+            ${PANDAS_VERSION+pandas=$PANDAS_VERSION}
```

Comment:
This line looks strange to me and not straight forward. I tested locally and it returns the right result, but why not applying the same pattern as above: `package=$PACKAGE_VERSION` ?